src/main/java/mchorse/blockbuster/client/particles/emitter/BedrockParticle.java

```diff
@@ -121,7 +123,13 @@ public void update(BedrockEmitter emitter)
 			/* Position */
 			Vector3f vec = new Vector3f(this.speed);
 			vec.scale(-(this.drag + this.dragFactor));
-
+			
```

Comment:
I just noticed some minor mistakes e.g. acceleration.y -= for every iteration... I'll change some things and add another feature.